src/modules/commentPopup.js

```diff
@@ -0,0 +1,82 @@
+import { fetchSingleProduct, getComments } from './commentAPI.js';
+import {
+  displayContent,
+  displayComments,
+  drawStars,
+  changeThumbnails,
+} from './displayPopup.js';
+import submitComment from './commentSubmit.js';
+import Counter from './count.js';
+
+const Count = new Counter();
+
```

Comment:
- [ ] Please, consider that there is still not correctly working of closing the popup. As you can see in the gif below, after closing the popup, it is still available and cover some space of the page. I mean, you can click on items for comment after that covering space. But all items under closed popup covering space are not active. Hope the gif will help to understand deeply the issue:

> This is when I tried to write a comment and close the popup. You can see the dark space after popup, I am showing with the cursor?
![2022-03-11 11 48 01](https://user-images.githubusercontent.com/49064106/157826085-412499c0-0985-48b2-86cb-78899872a955.gif)

> And here, you can see the reaction of the comment button of items under and outside of this dark space - you can not click on the comment button on items under dark space, but can click on items outside of this dark space:
![2022-03-11 12 00 30](https://user-images.githubusercontent.com/49064106/157826788-070b820d-d7dd-4147-b450-865b94090d51.gif)